packages/@lwc/template-compiler/src/__tests__/fixtures/directive-lwc-if-else/flattening/expected.js

```diff
@@ -5,17 +5,25 @@ const $fragment3 = parseFragment`<h1${3}>things</h1>`;
 const $fragment4 = parseFragment`<h1${3}>hello</h1>`;
 const $fragment5 = parseFragment`<h1${3}>world!</h1>`;
 function tmpl($api, $cmp, $slotset, $ctx) {
-  const { t: api_text, st: api_static_fragment, f: api_flatten } = $api;
-  return api_flatten([
+  const { t: api_text, st: api_static_fragment, fr: api_fragment } = $api;
```

Comment:
The original goal of this test was just to ensure that the returned values were flattened.  Since we're using fragments now I think we can rename it to `with-siblings` or something else.